cli/cluster.go

```diff
@@ -61,14 +72,26 @@ func runClusterAdd(args *docopt.Args) error {
 		GitHost: args.String["--git-host"],
 		TLSPin:  args.String["--tls-pin"],
 	}
-	if err := config.Add(s); err != nil {
+
+	if err := config.Add(s, args.Bool["--force"]); err != nil {
 		return err
 	}
+
+	setDefault := args.Bool["--default"] || len(config.Clusters) == 1
+
+	if setDefault && !config.SetDefault(s.Name) {
+		return errors.New("Cluster cannot be set as default.")
```

Comment:
This error is only shown if the cluster doesn't exist, right? If so, the message should reflect that.